tests/ui-toml/bool_comparison_equal/bool_comparison_equal.stderr

```diff
@@ -0,0 +1,52 @@
+error: inequality checks against true can be replaced with a comparison against `false`
+  --> $DIR/bool_comparison_equal.rs:7:13
+   |
+LL |     let _ = x != true;
+   |             ^^^^^^^^^ help: try simplifying it as shown: `x == false`
```

Comment:
Looking at the lint description, the lint was first desired to catch cases like `x == false` and suggest `!x` instead. I seems weird, that binary operators all of the sudden suggest such code. I think, suggesting `!x` directly would be the better option.